docs/getting-started/ios/screenshots.md

```diff
@@ -90,7 +92,7 @@ XCUIApplication *app = [[XCUIApplication alloc] init];
 [Snapshot setupSnapshot:app];
 [app launch];
 ```
-
+1. In the terminal run `fastlane snapshot`.  WARNING: Running the test in Xcode does not create the snapshots, is insufficient, and will not generate the correct results - although no tests will fail.  The command line program creates the necessary subdirectories, renames the files as appropriate, and generates the overview html page.
```

Comment:
The `1. In the terminal run `fastlane snapshot`.  WARNING: Runnin` does not carry on the numbering when code blocks are in the middle

<img width="1049" alt="screen shot 2018-03-14 at 7 58 17 am" src="https://user-images.githubusercontent.com/401294/37403664-94b898b0-275d-11e8-9472-6b8ecb146e4b.png">